hoomd/md/IntegratorTwoStep.h

```diff
@@ -20,14 +20,10 @@
 
     Notable design elements:
     - setDeltaT results in deltaT being set on all current integration methods
-    - to ensure that new methods also get set, addIntegrationMethod() also calls setDeltaT on the
-   method
-    - to interface with the python script, a removeAllIntegrationMethods() method is provided to
-   clear the list so they can be cleared and re-added from hoomd's internal list
+    - to interface with the python script, a the m_methods vectors is exposed with a list like API.
 
-    To ensure that the user does not make a mistake and specify more than one method operating on a
-   single particle, the particle groups are checked for intersections whenever a new method is added
-   in addIntegrationMethod()
+   TODO: ensure that the user does not make a mistake and specify more than one method operating on a
```

Comment:
Highlighting in case this TODO needs to be addressed now

Reply:
This is currently already a _problem_; however, in practice this is just a nice user warning for incorrect physics not a programming bug per se.